### PR TITLE
[NTOS:KE] Don't hardcode pool tag and TSS flag values.

### PR DESCRIPTION
**Pull request by HBelusca**
Don't hardcode pool tag and TSS flag values.

### CODE REVIEW

**Review by ThFabba — APPROVED** [review]

**Review by JoachimHenze — APPROVED** [review]
Approved. I did double-check that the replaced hardcodes do match what is currently defined for:
#define TAG_KERNEL                  '  eK'

But one question:
in i386/ketypes.h
#define I386_TSS                0x9
and in amd64/ketypes.h
#define I386_TSS                0x9
while what you replace here is
0b1001
Is that change intended to fix a mismatch?
Or do I miss something here?

**Comment by HBelusca** [comment]
>  0b1001
Is that change intended to fix a mismatch?
Or do I miss something here?

Hint: number representations (hexadecimal, binary, etc.)

**Comment by JoachimHenze** [comment]
Thx, have never seen the binary notation before in code. Only ever used hex or dec. Fine then. Learned something today. I do hope it's within the standard for a while and not some fancy recent adding of CXX11+.
If it's not bread-n-butter-C89-style, then it would be even better that you replace it here. :+1:

**Comment by HBelusca** [comment]
It's C99

**Review by DarkFire01 — APPROVED** [review]

**Review by binarymaster — APPROVED** [review]

**Comment by SergeGautherie** [comment]
"It's clear you spied on my #6014 :trollface: ,"

File: ntoskrnl/ke/i386/mproc.c [hunk 1]
```diff
@@ -146,11 +146,11 @@ KeStartAllProcessors(VOID)
...
     if (APInfo)
-        ExFreePoolWithTag(APInfo, '  eK');
+        ExFreePoolWithTag(APInfo, TAG_KERNEL);
     if (KernelStack)
         MmDeleteKernelStack(KernelStack, FALSE);
     if (DPCStack)
         MmDeleteKernelStack(DPCStack, FALSE);
 
-    DPRINT1("KeStartAllProcessors: Sucessful AP startup count is %u\n", ProcessorCount);
+    DPRINT1("KeStartAllProcessors: Successful AP startup count is %u\n", ProcessorCount);
```
[the remark this answers is not in the document]
Comment (SergeGautherie) [reply ntoskrnl/ke/i386/mproc.c:155]: Ah, ok.

**Pull request merged**